src/7-to-8/major-changes/cli.rst

```diff
@@ -42,6 +42,9 @@ renamed or removed.
   - We no longer include a command for locating this documentation.
```

Comment:
Commands like `cylc documentation` and `cylc checkpoint` that have been removed should say "removed" as the first sub-bullet point. (The intro says "changed or removed" but then the reader has to figure out which is which).

Reply:
I've split them into 3 lists:
- commands that have been removed entirely
- those which have been replaced
- those which have changed